crates/floresta-chain/src/pruned_utreexo/mod.rs

```diff
@@ -362,9 +362,9 @@ impl<T: BlockchainInterface> BlockchainInterface for Arc<T> {
         proof: Proof,
         inputs: HashMap<OutPoint, TxOut>,
         del_hashes: Vec<sha256::Hash>,
-        acc: Stump,
+        validate_block: bool,
     ) -> Result<(), Self::Error> {
-        T::validate_block(self, block, proof, inputs, del_hashes, acc)
+        T::validate_block(self, block, proof, inputs, del_hashes, validate_block)
     }
```

Comment:
Here, i included the verify_script arg

Reply:
directly in the API, the decision is consistent way down in the validate_block_transactions... maybe we could use a ref to get less overhead in his way down to the decision itself. 
another optimization would be setting the arg false when the bitcoin_consensus is not present